frontend/packages/integration-tests-cypress/OWNERS

```diff
@@ -0,0 +1,8 @@
+reviewers:
+  - rhamilto
+  - TheRealJon
+approvers:
+  - spadgett
+  - dtaylor113
+labels:
+  - e2e/cypress
```

Comment:
If we add a label, we'll need a corresponding openshift/release PR like https://github.com/openshift/release/pull/14656

Reply:
Hi @spadgett, do we want one to help with cypress + prow tracking?  (I'm not really sure what the label is used for)

Reply:
It's simply a label that gets applied by the bot to the PR. So any PR that changes files in this folder gets the label.

Reply:
I'd probably make this something like `kind/integration-tests` or maybe `kind/cypress` to match the current convention of label names.

Reply:
ok, updated to `kind/cypress` and I'll look into creating a corresponding openshift/release PR.

Reply:
Created openshift/release PR: https://github.com/openshift/release/pull/17118